test/e2e/tinkerbell_test.go

```diff
@@ -321,6 +321,72 @@ func TestTinkerbellKubernetes125UbuntuAddWorkerNodeGroupWithAPI(t *testing.T) {
 	)
 }
 
+func TestTinkerbellKubernetes125UbuntuTo126InPlaceUpgrade(t *testing.T) {
```

Comment:
Can we make the names a bit more descriptive so its more obvious from the name what exactly the test is doing
```suggestion
func TestTinkerbellKubernetesUbuntu2004InPlaceUpgrade125to126_1-CP_1-Worker(t *testing.T) {
```

Same thing would apply for all the tests here